sample-app/src/main/java/com/joinforage/android/example/ui/pos/POSFragment.kt

```diff
@@ -43,10 +46,13 @@ class POSFragment : Fragment() {
             object : ResultCallback<DeviceManager> {
                 override fun onFinish(deviceManager: DeviceManager) {
                     Log.i("CPay SDK", "DeviceManager created successfully")
+                    val terminalId = deviceManager.systemDevice.getSystemInfo(SystemDevice.SystemInfoType.SN)
+                    posViewModel.setTerminalId(terminalId)
```

Comment:
lmao - I feel like I just learned  shorthand way to do this from you. I was previously doing something longer